hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -305,6 +317,9 @@ private OMResponse submitRequestDirectlyToOM(OMRequest request) {
           OMAuditLogger.log(omClientRequest.getAuditBuilder());
           throw ex;
         }
+        if (ozoneManager.isLeaderExecutorEnabled()) {
```

Comment:
Would this mean that for v2 processing, single-mode OM will not be supported? If so, we should add a log here.